src/test/scala/chiseltest/tests/BundleLiteralsSpec.scala

```diff
@@ -11,7 +11,7 @@ import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
 class BundleLiteralsSpec extends AnyFlatSpec with ChiselScalatestTester with Matchers {
-  behavior of "Testers2"
+  behavior.of("Testers2")
```

Comment:
This is pretty bad, is there a rule to keep this as-is?

Reply:
I mad a PR in scalafmt for this https://github.com/scalameta/scalafmt/pull/2252